CONTRIBUTING.md

```diff
@@ -24,6 +24,23 @@ Check out our [README.md] for more details.
 
     * The [InnerSource Commons Slack] [#ispo-working-group] community will post the next meeting.
 
+## Our Artifacts
+
+In addition to our [Kanban board] cards, we produce the following artifacts.
+
+```mermaid
+graph LR
+    WG[ISPO Working Group] --> GD[GitHub Discussions]
+    WG --> IP[InnerSource Patterns]
+    WG --> MI[Managing InnerSource Projects]
+    WG --> DR[Google Drive Docs]
+```
+
+[GitHub Discussions] — A place to collect ideas for .
```

Comment:
Looks like this sentence isn't complete yet, is it?

Reply:
Also did you want to render all of these as a list?

Reply:
Not complete, it's a point of discussion.

What are the use cases? :)

Reply:
> Also did you want to render all of these as a list?

I kept it raw for now, but I'm not opposed to it.

Reply:
> > Looks like this sentence isn't complete yet, is it?
> 
> 
> 
> Not complete, it's a point of discussion.
> 
> 
> 
> What are the use cases? :)

Got it. I missed that this PR was still WIP, so probably commented too early.